### PR TITLE
feat: add diagnostic tool

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: python/openmldb_tool/diagnostic_tool/collector.py [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+def buf2str(buf: BufferedFile) -> str:
+    return buf.read().decode("utf-8")
+
+
+class Collector:
+    def __init__(self, dist_conf: DistConf):
+        self.dist_conf = dist_conf
+        # use one ssh client to connect all servers, ssh connections won't keep alive
+        self.ssh_client = paramiko.SSHClient()
+        self.ssh_client.load_system_host_keys()
```
Comment (vagetablechicken) [inline python/openmldb_tool/diagnostic_tool/collector.py:53]: add the policy
```suggestion
        self.ssh_client.load_system_host_keys()
        self.ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

**Comment by lumianph** [comment]
can we also add the doc for this tool? @dl239

**Review by vagetablechicken — APPROVED** [review]

File: src/sdk/CMakeLists.txt [hunk 2]
```diff
@@ -103,23 +103,24 @@ if(SQL_PYSDK_ENABLE)
...
-            COMMAND cd ${PROJECT_SOURCE_DIR}/python/ && ${Python3_EXECUTABLE} setup.py bdist_wheel --plat-name ${PYTHON_PLATFORM}
+            COMMAND cd ${PROJECT_SOURCE_DIR}/python/openmldb_sdk/ && ${Python3_EXECUTABLE} setup.py bdist_wheel --plat-name ${PYTHON_PLATFORM}
+            COMMAND cd ${PROJECT_SOURCE_DIR}/python/openmldb_tool/ && ${Python3_EXECUTABLE} setup.py bdist_wheel
             BYPRODUCTS
-                python/build
-                python/dist
-                python/sqlalchemy-openmldb.egg-info)
+                build
+                dist
+                sqlalchemy-openmldb.egg-info)
```
Comment (aceforeverd) [inline src/sdk/CMakeLists.txt]: no sqlalchemy prefix

File: src/sdk/CMakeLists.txt [hunk 3]
```diff
@@ -103,23 +103,24 @@ if(SQL_PYSDK_ENABLE)
...
+            COMMAND ${CMAKE_COMMAND} -E copy  ${CMAKE_BINARY_DIR}/sql_pysdk/openmldb/sql_router_sdk.py ${PROJECT_SOURCE_DIR}/python/openmldb_sdk/openmldb/native/
             COMMENT "copy generated native library and sql_router_sdk python file to python project"
-            COMMAND cd ${PROJECT_SOURCE_DIR}/python/ && ${Python3_EXECUTABLE} setup.py bdist_wheel --plat-name ${PYTHON_PLATFORM}
+            COMMAND cd ${PROJECT_SOURCE_DIR}/python/openmldb_sdk/ && ${Python3_EXECUTABLE} setup.py bdist_wheel --plat-name ${PYTHON_PLATFORM}
+            COMMAND cd ${PROJECT_SOURCE_DIR}/python/openmldb_tool/ && ${Python3_EXECUTABLE} setup.py bdist_wheel
             BYPRODUCTS
-                python/build
-                python/dist
-                python/sqlalchemy-openmldb.egg-info)
+                build
```
Comment (aceforeverd) [inline src/sdk/CMakeLists.txt]: guess relative path is not correct ? add the `${PROJECT_SOURCE_DIR}/python/openmldb_sdk` should fix

**Review by aceforeverd — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**